tfhe/src/integer/gpu/server_key/radix/sub.rs

```diff
@@ -250,7 +250,7 @@ impl CudaServerKey {
     ///
     /// - `stream` __must__ be synchronized to guarantee computation has finished, and inputs must
     ///   not be dropped until stream is synchronised
-    pub unsafe fn sub_assign_async<T: CudaIntegerRadixCiphertext>(
+    pub unsafe fn sub_and_propagate_async<T: CudaIntegerRadixCiphertext>(
```

Comment:
This is the `sub_async` function, it should call directly the entry point of integer/gpu/mod.rs